src/cxx/basic_api.cc

```diff
@@ -468,6 +468,32 @@ JNIEXPORT void JNICALL Java_foundation_pEp_jniadapter_Engine__1config_1passphras
     }
 }
 
+JNIEXPORT void JNICALL Java_foundation_pEp_jniadapter_Engine__1config_1passphrase_1for_1new_1keys_1by_1email(JNIEnv *env,
+                                                                                                  jobject obj,
+                                                                                                  jboolean enable,
+                                                                                                  jbyteArray email,
+                                                                                                  jbyteArray passphrase)
```

Comment:
That's one weird style of indentation (just saying, this is just cosmetics).

Reply:
Updated to match the style of the file more or less.